src/main/java/network/brightspots/rcv/Utils.java

```diff
@@ -16,9 +16,31 @@
 
 package network.brightspots.rcv;
 
+import java.util.List;
+
 class Utils {
 
   static boolean isNullOrBlank(String s) {
     return s == null || s.isBlank();
   }
+
+  static String listToSentenceWithQuotes(List<String> list) {
```

Comment:
not really a "sentence" but the attention to detail here more than makes up for it

Reply:
Good point... but other languages call this a "sentence" for some reason, and who am I to think that I'm better than they are?

https://apidock.com/rails/Array/to_sentence